projects/kyrgyzstan/.env

```diff
@@ -0,0 +1 @@
+DATA_DIR=./data
```

Comment:
please do not remove the `COMPOSE_PROJECT_NAME=pelias` line as per other projects.

Reply:
Hello it's been a long time, since I added DATA_DIR line, I looked in other countires project and I see a different pattern instead of `DATA_DIR=./data` it is `DATA_DIR=/tmp/pelias/countryname` so I updated it too. Hope it will not break anything.